src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -220,7 +219,7 @@ public boolean processAutoDiscovery(byte[] buffer) {
       String[] discovered;
 
       try {
-        String configs = new String(buffer, CharEncoding.UTF_8);
+        String configs = new String(buffer, "UTF-8");
```

Comment:
Good catch to remove this dependency since it was only used for this. 
What about creating constant instead of having duplicated strings?

Reply:
I can do that if that's what you'd prefer.